apiserver/facades/client/cloud/cloud.go

```diff
@@ -28,6 +28,12 @@ import (
 
 var logger = loggo.GetLogger("juju.apiserver.cloud")
 
+// CloudV5 defines the methods on the cloud API facade, version 5.
```

Comment:
I think I get why we do it, but it's confusing that the interface for version N embeds version N-1 while the struct for version N embeds version N+1. Maybe it would be clearer for the interfaces to just be explicit and list all of the methods?